shared/api/image_transforms_phi_3.hpp

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: as our list of processor files slowly grows, should we perhaps group the files inside shared/api? perhaps into `c_api`, `imaging`, and `tokenizing` - we can do this organizational work in a future PR.

Reply:
After audio C abi, we may revisit it and re-org the folder if needed